netutils/netlib/netlib_ipv6route.c

```diff
@@ -131,9 +131,12 @@ ssize_t netlib_read_ipv6route(FILE *stream,
       return 0;
     }
 
-  /* The first line of the group should consist of a number index */
+  /* The first non-space char of the first line of the group should consist of a number index */
+  int idx = 0;
```

Comment:
declare at the top of the function

Reply:
Got it.  The second commit in the PR addresses this.